packages/core/src/codewhisperer/activation.ts

```diff
@@ -191,8 +191,6 @@ export async function activate(context: ExtContext): Promise<void> {
         }),
         // show introduction
         showIntroduction.register(),
-        // direct CodeWhisperer connection setup with customization
-        connectWithCustomization.register(),
```

Comment:
nit:

        // direct CodeWhisperer connection setup with customization
        connectWithCustomization.register(),
        connectWithCustomization()?.register() ?? ...[],

Reply:
this won't work. any object being pushed must have dispose() property